scripts/select_appointments.sql

```diff
@@ -1 +1,25 @@
 -- retrieve appointment information
+SELECT
+p.given_name,
+p.family_name,
+a.time
+
+FROM patients p
+  JOIN appointments a ON a.patient_id = p.id
+  JOIN doctors d ON d.id = a.doctor_id
+
+WHERE d.family_name = 'Hubbard'
+;
+
+SELECT
+d.given_name,
+d.family_name,
+a.time
+
+FROM patients p
+  JOIN appointments a ON a.patient_id = p.id
+  JOIN doctors d ON d.id = a.doctor_id
+
+WHERE p.given_name = 'Marsha'
+AND p.family_name = 'Wilcox'
+;
```

Comment:
This didn't retrieve the correct entries from the tables. Here's some code that does:

```sql
-- retrieve appointment information
-- for Dr. Hubbard
SELECT p.family_name, a.date, a.time
  FROM patients p
  INNER JOIN appointments a
    ON a.patient_id = p.id
  INNER JOIN doctors d
    ON a.doctor_id = d.id
WHERE d.family_name = 'Hubbard';
-- for patient Wilcox
SELECT d.family_name, a.date, a.time
  FROM doctors d
  INNER JOIN appointments a
    ON a.doctor_id = d.id
  INNER JOIN patients p
    ON a.patient_id = p.id
WHERE p.family_name = 'Wilcox';
```